azerothcore/azerothcore-wotlk
# fix(Core/Spells): Spell [Pounce Bleed] should not be removed by Cloak of Shadows.

## Changes Proposed:
-  Cloak of Shadows: Spell [Pounce Bleed] can no longer be removed with Cloak of Shadows.


## Issues Addressed:
- Closes 



## SOURCE:



## Tests Performed:

- Compile without errors in windows.

## How to Test the Changes:

- Log in with a druid and a rogue, become an invisible cat and cast ``[Pounce]`` on the rogue, then try to remove the bleed effect with a ``[Cloak of shadows]`` and you will find that you can no longer remove the bleed effect.

## Known Issues and TODO List:



## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
